crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -3319,6 +3323,10 @@ type StakedSui implements IMoveObject & IObject & IOwner {
 	"""
 	requestedEpoch: Epoch
 	"""
+	The id of the validator staking pool this stake belongs to.
```

Comment:
nit: (Just to make clear what kind of ID -- in GraphQL there is a different concept of `ID` that we have not implemented yet, but probably will eventually).

```suggestion
	The Object ID of the validator staking pool this stake belongs to.
```